CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## Unreleased
+
+- Add null check to beginning of `sanitizeUrl` function
```

Comment:
I would just reference the issue this closes at the end of the line like => (#54). That should automatically link to the issue as well. Otherwise LGTM